dist/angular-patternfly.js

```diff
@@ -1670,6 +1683,7 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", "$window
        }
 
        $scope.data = {
+         dataAvailable: true,
```

Comment:
> @simon3z the example will not work without it :-)

@yaacov it doesn't sound like an explanation on why it's required. You're just making more evident that there's some code relying on an extra unneeded flag.